### ISSUE TITLE
iOS version?

### ISSUE THREAD

**Issue by asher-pembroke** [shashi278/social-auth-kivy#1]
Wondering if a similar procedure would work for iOS? 

Thanks

**Comment by shashi278** [comment]
@asher-pembroke I've never used an iOS and have never compiled for the same so I can't say much about it. But it doesn't seem impossible to me 🤔

**Comment by Python3-8** [comment]
This module won't work on iOS because it uses pyjnius (for Java classes), not pyobjus (I think for Objective-C classes).

**Comment by shashi278** [comment]
@Python3-8 you're right! Also there are detailed docs about integrating social auth in iOS apps either using firebase or native way. So if anyone who has earlier worked with pyobjus and Obj-C can make it for iOS without much difficulty.
